<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Lenovo-PC" ignored="0" name="Test2" tests="2" failures="1" timestamp="18 May 2020 02:18:39 GMT" time="88.883" errors="0">
  <testcase name="x" time="14.072" classname="com.org.mypac.Mytest"/>
  <testcase name="y" time="0.001" classname="com.org.mypac.Mytest">
    <failure type="java.lang.AssertionError" message="y() failed to execute expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: y() failed to execute expected [true] but found [false]
at com.org.mypac.Mytest.y(Mytest.java:115)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- y -->
</testsuite> <!-- Test2 -->
